VSCode devcontainer error during creation 'mapiterinit redeclared in this block'

### Describe the bug

When I do a fresh clone of the repository and open the directory with VSCode, it will ask me to open the repository in a development container. When I do this, I'll receive an error during setup of the container and the build fails.

The error is:

```
# runtime
/usr/local/go/src/runtime/map.go:877:6: mapiterinit redeclared in this block
        /usr/local/go/src/runtime/linkname_swiss.go:91:6: other declaration of mapiterinit
/usr/local/go/src/runtime/map.go:937:6: mapiternext redeclared in this block
        /usr/local/go/src/runtime/linkname_swiss.go:144:6: other declaration of mapiternext
/usr/local/go/src/runtime/map.go:1519:6: reflect_mapiterinit redeclared in this block
        /usr/local/go/src/runtime/linkname_swiss.go:123:6: other declaration of reflect_mapiterinit
/usr/local/go/src/runtime/map.go:1536:6: reflect_mapiternext redeclared in this block
        /usr/local/go/src/runtime/linkname_swiss.go:173:6: other declaration of reflect_mapiternext
/usr/local/go/src/runtime/map.go:1550:6: reflect_mapiterkey redeclared in this block
        /usr/local/go/src/runtime/linkname_swiss.go:191:6: other declaration of reflect_mapiterkey
/usr/local/go/src/runtime/map.go:1564:6: reflect_mapiterelem redeclared in this block
        /usr/local/go/src/runtime/linkname_swiss.go:209:6: other declaration of reflect_mapiterelem
/usr/local/go/src/runtime/map_fast32_swiss.go:18:6: mapaccess1_fast32 redeclared in this block
        /usr/local/go/src/runtime/map_fast32.go:13:6: other declaration of mapaccess1_fast32
/usr/local/go/src/runtime/map_fast32_swiss.go:29:6: mapaccess2_fast32 redeclared in this block
        /usr/local/go/src/runtime/map_fast32.go:62:6: other declaration of mapaccess2_fast32
/usr/local/go/src/runtime/map_fast32_swiss.go:41:6: mapassign_fast32 redeclared in this block
        /usr/local/go/src/runtime/map_fast32.go:113:6: other declaration of mapassign_fast32
/usr/local/go/src/runtime/map_fast32_swiss.go:52:6: mapassign_fast32ptr redeclared in this block
        /usr/local/go/src/runtime/map_fast32.go:212:6: other declaration of mapassign_fast32ptr
/usr/local/go/src/runtime/map_fast32_swiss.go:52:6: too many errors
make: *** [Makefile:40: install] Error 1
[31155 ms] postCreateCommand from devcontainer.json failed with exit code 2. Skipping any further user-provided commands.
```

I traced the error already and will post an PR:

The base image of the dev container is `mcr.microsoft.com/devcontainers/go:1-1.23-bookworm` which contains go in version 1.23. Then during setup of the container `make install` is called which will update go to version 1.24. The culprit lies in the added go feature for vscode in `devcontainer.json`:  

```
	"features": {
                ...
		"ghcr.io/devcontainers/features/go:1": {
			"version": "latest"
		}
	},
```

This seem to install a second version of go inside the container which conflicts with the already installed go version. When I remove the go feature from `devcontainer.json` everything works as expected.

### Steps to reproduce

1. Open repo with VSCode
2. Click on "Open in Container"
3. Error



### Configuration details

```yaml
Does not apply here
```

### Log details

```text
Does not apply here
```

### What type of operating system or environment does evcc run on?

Linux

### External automation

- [x] I have made sure that no external automation like HomeAssistant or Node-RED is active or accessing any of the mentioned devices when this issue occurs.

### Nightly build

- [x] I have verified that the issue is reproducible with the latest nightly build

### Version

_No response_

--- a/.devcontainer/devcontainer.json
+++ b/.devcontainer/devcontainer.json
@@ -17,9 +17,6 @@
 			"version": "lts",
 			"pnpmVersion": "latest",
 			"nvmVersion": "latest"
-		},
-		"ghcr.io/devcontainers/features/go:1": {
-			"version": "latest"
 		}
 	},
 	"postCreateCommand": "make install-ui && make install",

